### PR TITLE
DOCSP-13961: query operators fixes

### PR DESCRIPTION
**Pull request by ccho-mongodb**
JIRA:
https://jira.mongodb.org/browse/DOCSP-13961

Staging:
https://docs-mongodbcom-staging.corp.mongodb.com/be2f69a/node/docsworker-xlarge/DOCSP-13961-evaluation-operators-fix/fundamentals/crud/query-document

### CODE REVIEW

**Review by terakilobyte — CHANGES_REQUESTED** [review]
Good changes overall. I left one nitpick/concern. Could you also look at the code blocks ~141?

```js
collection.find({
  rating: { $eq: 5 },	     
  rating: { $eq: 5 },
  qty: { $gt: 4 }
}
``` 

is missing a paren and should be

```js
collection.find({
  rating: { $eq: 5 },	     
  rating: { $eq: 5 },
  qty: { $gt: 4 }
})
``` 

and 

```js
collection.find({
  $and : [	      
    rating: { $eq: 5 },
    qty: { $gt: 4 }
  ]
}
```

is missing a paren and has invalid syntax. It should be

```js
collection.find({
  $and : [	      
    { rating: { $eq: 5 } },
    { qty: { $gt: 4 } }
  ]
})
```

File: source/fundamentals/crud/query-document.txt [hunk 1]
```diff
@@ -7,16 +7,16 @@ Specify a Query
...
-a collection; the query document allows you to make your searches more
-specific. Query documents contain one or more field-level expressions
-that determine which documents to include in the result set. You can
-match fields against literal values (e.g. ``{title:'The Room'}``) or
-compose :manual:`query operators </reference/operator/query/>` to
-express more complex queries. MongoDB
-supports multiple types of query operator, including:
+Most CRUD operations allow you to narrow the set of matched documents by
+specifying matching criteria in a **query document**. Query documents contain
+one or more field-level expressions that determine which documents to include
```
[the remark this answers is not in the document]
Comment (ccho-mongodb) [reply source/fundamentals/crud/query-document.txt]: Ok, I removed the term which came from the original text of this doc. Let me know if it looks ok or whether you have a different recommendation.

File: source/fundamentals/crud/query-document.txt [hunk 2]
```diff
@@ -7,16 +7,16 @@ Specify a Query
...
-a collection; the query document allows you to make your searches more
-specific. Query documents contain one or more field-level expressions
-that determine which documents to include in the result set. You can
-match fields against literal values (e.g. ``{title:'The Room'}``) or
-compose :manual:`query operators </reference/operator/query/>` to
-express more complex queries. MongoDB
-supports multiple types of query operator, including:
+Most CRUD operations allow you to narrow the set of matched documents by
+specifying matching criteria in a **query document**. Query documents contain
+one or more expressions that apply to specific fields which  determine which
```
Comment (terakilobyte) [inline source/fundamentals/crud/query-document.txt:12]:
```suggestion
one or more query operators that apply to specific fields which  determine which
```

**Review by terakilobyte — APPROVED** [review]
I prefer the overall term "query operators" and that aligns with the page contents, but I won't let this be a blocker on a page that's been out in the wild for a bit now.

**Pull request merged**